xen-exporter.py

```diff
@@ -22,7 +23,10 @@ def lookup_vm_name(vm_uuid, session):
 
 
 def lookup_sr_name_by_uuid(sr_uuid, session):
-    return session.xenapi.SR.get_name_label(session.xenapi.SR.get_by_uuid(sr_uuid))
+    try:
+       return session.xenapi.SR.get_name_label(session.xenapi.SR.get_by_uuid(sr_uuid))
+    except:
```

Comment:
Why is this ignoring errors now?

What error is happening?

I'd be alright with this, but it should be `except Exception` and then should print out the error that happened.

Reply:
If we cannot find the name label for a UUID that seems rather unusual. Not sure how that could happen.

But anyway, let's make the `except` more specific to that xen failure. Log the exception message, and then return from the method with the UUID instead of `None` which is what it will be returning right now if there's an error.